### PR TITLE
[VSPC-11] added menu tab in header with link to the staff view, post

### PR DESCRIPTION
**Pull request by setushah**
login

### CODE REVIEW

File: vspace/pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
 		<db.url>jdbc:mysql://localhost:3306/vspace</db.url>
 		<db.user>vspace</db.user>
 		<db.password>vspace</db.password>
-		<uploaded.files.path>/Users/jdamerow/Software/VirtualSpaces2.0/uploads/</uploaded.files.path>
+		<uploaded.files.path>/Users/setushah/Documents/uploads/</uploaded.files.path>
```
Comment (jdamerow) [inline vspace/pom.xml]: please do not commit this file.

File: vspace/src/main/webapp/WEB-INF/tiles/skeleton_exhibition.jsp [hunk 2]
```diff
@@ -41,6 +41,11 @@
             <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
           </li>
           
+          <sec:authorize access="isAuthenticated()">
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/tiles/skeleton_exhibition.jsp]: this should only be shown for users with role 'STAFF'.

**Comment by jdamerow** [comment]
please resolve conflicts

**Comment by namrathaov** [comment]
Reverted the pom.xml file and made the requested changes.

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
retest this please

**Comment by jdamerow** [comment]
retest this please

**Comment by jdamerow** [comment]
retest this please

**Comment by jdamerow** [comment]
retest this please

File: vspace/src/main/webapp/WEB-INF/tiles/skeleton_exhibition.jsp [hunk 3]
```diff
@@ -40,7 +40,13 @@
           <li class="nav-item active">
             <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
           </li>
-          
+          <sec:authorize access="hasRole('ROLE_STAFF')">
+		<li class="nav-item active">
+			<a class="nav-link href="<c:url value="/staff/space/list" />">Menu
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/tiles/skeleton_exhibition.jsp]: there is a quotation mark missing. this does not generate a correct link.

**Comment by namrathaov** [comment]
Changed the quote and verified the link is working.

File: vspace/src/main/webapp/WEB-INF/tiles/skeleton_exhibition.jsp (hunk 3 above)
Comment (jdamerow) [reply vspace/src/main/webapp/WEB-INF/tiles/skeleton_exhibition.jsp]: still true

**Pull request closed without merging**